assignments/prototypes.js

```diff
@@ -32,7 +52,21 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
- 
+ function Humanoid(attributes){
+  this.team = attributes.team;
+  this.weapons = attributes.weapons;
+  this.language = attributes.language;
+  this.healthPoints = attributes.healthPoints;
+  this.createdAt = attributes.createdAt;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+ }
```

Comment:
You shouldn't have all these properties explicitly written, but inheriter until this point from the parent and grandparent since the humanoid should be using 
```js
 CharacterStats.call(this, attributes)
```
to start and inherit a majority of those attributes so your `Humanoid` constructor should look like this:
```js
function Humanoid(attributes) {
	CharacterStats.call(this, attributes);
	this.team = attributes.team;
	this.weapons = attributes.weapons;
	this.language = attributes.language;
}
```

While letting the inheritance from the parent(CharacterStats) who is inheriting some its properties from the grandparent(GameObject). So through the inheritance you shouldn't need to explicitly specify them all.